bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManagerImpl.java

```diff
@@ -625,13 +640,15 @@ private void initializeHandler(Thing thing) {
             }
             ThingType thingType = getThingType(thing);
             applyDefaultConfiguration(thing, thingType);
+
             if (isInitializable(thing, thingType)) {
                 setThingStatus(thing, buildStatusInfo(ThingStatus.INITIALIZING, ThingStatusDetail.NONE));
                 doInitializeHandler(thing.getHandler());
             } else {
-                logger.debug("Thing '{}' not initializable, check required configuration parameters.", thing.getUID());
-                setThingStatus(thing,
-                        buildStatusInfo(ThingStatus.UNINITIALIZED, ThingStatusDetail.HANDLER_CONFIGURATION_PENDING));
+                logger.debug("Thing '{}' not initializable, check required configuration parameters.",
+                        thing.getUID());
+                setThingStatus(thing, buildStatusInfo(ThingStatus.UNINITIALIZED,
+                        ThingStatusDetail.HANDLER_CONFIGURATION_PENDING));
```

Comment:
lines 648 - 651 do not seem to have been changed. please restore the original format.
it seems your formatter is not ESH default, please check and adopt.